src/main/java/com/google/sps/app/servlets/NewTripServlet.java

```diff
@@ -120,17 +121,17 @@ public Trip getTrip(HttpServletRequest request){
         Date end = DataStoreHelper.parseInputDate(request.getParameter(END_DATE_PARAM));
         String participantInput = StringEscapeUtils.escapeHtml4(request.getParameter(PARTICIPANTS_PARAM));
         List<String> participantIds = new ArrayList<String>();
-        List<String> checkerduplicate = new ArrayList<String>();
+        List<String> names = new ArrayList<String>();
```

Comment:
Can you name these existingParticipants and newParticipants. It is slightly confusing currently.